EU4toV2/Source/V2World/Pops/PopMapper.h

```diff
@@ -0,0 +1,22 @@
+#ifndef POP_MAPPER_H
+#define POP_MAPPER_H
+
+#include "newParser.h"
+#include "PopType.h"
+#include <map>
+
+namespace mappers
+{
+	class PopMapper : commonItems::parser
+	{
+	public:
+		PopMapper() = default;
+		PopMapper(std::istream& theStream);
+		std::map<int, PopType> getPopMap() const { return popMap; }
+
+	private:
+		std::map<int, PopType> popMap;
```

Comment:
It's unclear this is meant to map provinces to pops. Might want to choose a more clear name.

Reply:
fixed